src/authConfig.js

```diff
@@ -20,6 +21,7 @@ export const msalConfig = {
         cacheLocation: "sessionStorage", // This configures where your cache will be stored
         storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
     },
+// </ms_docref_msal_configuration>
```

Comment:
This should be at the bottom of the block to include the closing brace. We can remove the logger config from this sample if this was undesirable from a docs perspective

Reply:
Hello @tnorling that would be great, the goal is to ensure what we currently have in the docs remains exactly the same when we implement this solution